doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,209 @@
+     However, you may choose to switch this off for a specific method, class, or the entire assembly.
+
+     This can be achieved using the `ImplicitCommands` attribute. Here are some examples:
+
+     ```csharp
+     [assembly:ImplicitCommands(false)]
+     
+     [ImplicitCommands(true)]
+     public partial record MyModel();
+
+     [ImplicitCommands(true)]
+     public void MyCommand();
+     ```
+
+     These attributes can be used interchangeably to control command generation, for example, you may choose to disable the implicit feed command parameter altogether by using the attribute with the `assembly:` prefix. You can then opt-in for specific classes or methods. You can also switch off specific classes while opting-in certain methods by decorating them with this attribute as enabled. You can choose any combination to opt-in or out of implicit command generation.
+
+     #### FeedParameter attribute
+
+     You can explicitly match a parameter with a Feed even if the names don't match by decorating the parameter with the `FeedParameter` attribute:  
+     
+     ```csharp
+     public IFeed<string> Message { get; }
+     
+     public async ValueTask Share([FeedParameter(nameof(Message))] string msg)
+     {
+     }
+     ```
+
+     #### ImplicitFeedCommandParameter attribute
+
+     You can also opt in or out of implicit matching of Feeds and command parameters by decorating the current assembly or class with the `ImplicitFeedCommandParameters` attribute:  
+     
+     ```csharp
+     [assembly:ImplicitFeedCommandParameter(false)]
+     
+     [ImplicitFeedCommandParameter(true)]
+     public partial record MyModel
+     ```
+
+     Like `ImplicitCommands`, these can also be nested to opt-in or out sections specific sections of the app.
+
+1. Using factory methods
+
+    Adding Commands via code generation is sufficient enough to probably cover all scenarios. However, sometimes you'd want to fine-grain your Commands and declare them in an explicit manner.  
+    Commands can be built manually using the static class [`Command`](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Presentation/Commands/Command.cs).  
+    This class provides factory methods for creating commands.
+
+    - Async
+
+    ```csharp
+    public ICommand MyCommand => Command.Async(async(ct) => await PingServer(ct));
+    ```
+
+    > [!Note]  
+    > You need not worry about the `=>` operator and that the `Command.Async` will be called over and over, this value is cached and will only be called once.  
+    The benefit of using `=>` in place of traditional `{ get; } = ...`, is that local methods are accessible (otherwise these would have been required to be initialized in the constructor).
+
+    - `Create` & `Create<T>`;
+
+    To create a command you can use the fluent API of `ICommandBuilder` provided in the `Command.Create` factory methods, which provides the following three methods:
```

Comment:
This sentence only made sense after I'd read the following section (down to the diagram) several times!